hack/gotest.sh

```diff
@@ -35,7 +35,12 @@ else
     JQ_FILTER='select(has("Output") and (.Action=="output") and (has("Test")|not) and (.Output!="PASS\n") and (.Output!="FAIL\n") and (.Output|startswith("coverage:")|not) and (.Output|contains("[no test files]")|not)) | .Output'
 fi
 
-echo "go test $@"
+if [[ " ${@}" =~ "pkg/skaffold" ]]; then
```

Comment:
I don't understand why we need to change this echo...what purpose does this serve?

Reply:
The `echo go test $@` list all the packages on this branch which is really long other wise. 
before  On master
```
tejaldesai@@skaffold (add_diag_pkg)$ make test
go test -count=1 -race -short -timeout=90s ./...
```

On branch without the check:
```
tejaldesai@@skaffold (add_diag_pkg)$ make test
go test -count=1 -race -short -timeout=90s github.com/GoogleContainerTools/skaffold/cmd/skaffold github.com/GoogleContainerTools/skaffold/cmd/skaffold/app github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd/config github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd/credits github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd/credits/statik github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd/schema github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd/statik github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/flags github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/tips github.com/GoogleContainerTools/skaffold/examples/bazel github.com/GoogleContainerTools/skaffold/examples/compose github.com/GoogleContainerTools/skaffold/examples/gcb-kaniko github.com/GoogleContainerTools/skaffold/examples/generate-pipeline github.com/GoogleContainerTools/skaffold/examples/getting-started github.com/GoogleContainerTools/skaffold/examples/google-cloud-build github.com/GoogleContainerTools/skaffold/examples/kaniko github.com/GoogleContainerTools/skaffold/examples/microservices/leeroy-app github.com/GoogleContainerTools/skaffold/examples/microservices/leeroy-web github.com/GoogleContainerTools/skaffold/examples/profile-patches/base-service github.com/GoogleContainerTools/skaffold/examples/profile-patches/hello-service github.com/GoogleContainerTools/skaffold/examples/profile-patches/world-service github.com/GoogleContainerTools/skaffold/examples/profiles/hello-service github.com/GoogleContainerTools/skaffold/examples/profiles/world-service github.com/GoogleContainerTools/skaffold/examples/structure-tests github.com/GoogleContainerTools/skaffold/examples/tagging-with-environment-variables github.com/GoogleContainerTools/skaffold/hack/man github.com/GoogleContainerTools/skaffold/hack/schemas github.com/GoogleContainerTools/skaffold/hack/versions/cmd/latest github.com/GoogleContainerTools/skaffold/hack/versions/cmd/latest_released github.com/GoogleContainerTools/skaffold/hack/versions/cmd/mark_latest_released github.com/GoogleContainerTools/skaffold/hack/versions/cmd/new github.com/GoogleContainerTools/skaffold/hack/versions/cmd/schema_check github.com/GoogleContainerTools/skaffold/hack/versions/cmd/update_comments github.com/GoogleContainerTools/skaffold/hack/versions/pkg/diff github.com/GoogleContainerTools/skaffold/hack/versions/pkg/schema github.com/GoogleContainerTools/skaffold/integration github.com/GoogleContainerTools/skaffold/integration/examples/bazel github.com/GoogleContainerTools/skaffold/integration/examples/compose github.com/GoogleContainerTools/skaffold/integration/examples/gcb-kaniko github.com/GoogleContainerTools/skaffold/integration/examples/generate-pipeline github.com/GoogleContainerTools/skaffold/integration/examples/getting-started github.com/GoogleContainerTools/skaffold/integration/examples/google-cloud-build github.com/GoogleContainerTools/skaffold/integration/examples/kaniko github.com/GoogleContainerTools/skaffold/integration/examples/microservices/leeroy-app github.com/GoogleContainerTools/skaffold/integration/examples/microservices/leeroy-web github.com/GoogleContainerTools/skaffold/integration/examples/profile-patches/base-service github.com/GoogleContainerTools/skaffold/integration/examples/profile-patches/hello-service github.com/GoogleContainerTools/skaffold/integration/examples/profile-patches/world-service github.com/GoogleContainerTools/skaffold/integration/examples/profiles/hello-service github.com/GoogleContainerTools/skaffold/integration/examples/profiles/world-service github.com/GoogleContainerTools/skaffold/integration/examples/structure-tests github.com/GoogleContainerTools/skaffold/integration/examples/tagging-with-environment-variables github.com/GoogleContainerTools/skaffold/integration/skaffold github.com/GoogleContainerTools/skaffold/pkg/skaffold/apiversion github.com/GoogleContainerTools/skaffold/pkg/skaffold/build github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/bazel github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/buildpacks github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/cache github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/cluster github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/custom github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/gcb github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/jib github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/local github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/misc github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/tag github.com/GoogleContainerTools/skaffold/pkg/skaffold/color github.com/GoogleContainerTools/skaffold/pkg/skaffold/config github.com/GoogleContainerTools/skaffold/pkg/skaffold/constants github.com/GoogleContainerTools/skaffold/pkg/skaffold/debug github.com/GoogleContainerTools/skaffold/pkg/skaffold/deploy github.com/GoogleContainerTools/skaffold/pkg/skaffold/deploy/kubectl github.com/GoogleContainerTools/skaffold/pkg/skaffold/deploy/resource github.com/GoogleContainerTools/skaffold/pkg/skaffold/docker github.com/GoogleContainerTools/skaffold/pkg/skaffold/event github.com/GoogleContainerTools/skaffold/pkg/skaffold/filemon github.com/GoogleContainerTools/skaffold/pkg/skaffold/gcp github.com/GoogleContainerTools/skaffold/pkg/skaffold/generate_pipeline github.com/GoogleContainerTools/skaffold/pkg/skaffold/initializer github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubectl github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/context github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/portforward github.com/GoogleContainerTools/skaffold/pkg/skaffold/pipeline github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner/runcontext github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner/util github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/defaults github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/util github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1alpha1 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1alpha2 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1alpha3 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1alpha4 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1alpha5 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta1 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta10 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta11 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta12 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta13 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta14 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta15 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta16 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta17 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta2 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta3 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta4 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta5 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta6 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta7 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta8 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1beta9 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v2alpha1 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v2alpha2 github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/validation github.com/GoogleContainerTools/skaffold/pkg/skaffold/server github.com/GoogleContainerTools/skaffold/pkg/skaffold/sources github.com/GoogleContainerTools/skaffold/pkg/skaffold/survey github.com/GoogleContainerTools/skaffold/pkg/skaffold/sync github.com/GoogleContainerTools/skaffold/pkg/skaffold/test github.com/GoogleContainerTools/skaffold/pkg/skaffold/test/structure github.com/GoogleContainerTools/skaffold/pkg/skaffold/trigger github.com/GoogleContainerTools/skaffold/pkg/skaffold/update github.com/GoogleContainerTools/skaffold/pkg/skaffold/util github.com/GoogleContainerTools/skaffold/pkg/skaffold/version github.com/GoogleContainerTools/skaffold/pkg/skaffold/warnings github.com/GoogleContainerTools/skaffold/pkg/skaffold/yamltags github.com/GoogleContainerTools/skaffold/pkg/webhook/constants github.com/GoogleContainerTools/skaffold/pkg/webhook/gcs github.com/GoogleContainerTools/skaffold/pkg/webhook/github github.com/GoogleContainerTools/skaffold/pkg/webhook/kubernetes github.com/GoogleContainerTools/skaffold/pkg/webhook/labels github.com/GoogleContainerTools/skaffold/proto github.com/GoogleContainerTools/skaffold/testutil github.com/GoogleContainerTools/skaffold/webhook
warning: GOPATH set to GOROOT (/usr/local/go) has no effect
?   	github.com/GoogleContainerTools/skaffold/cmd/skaffold	[no test files]
=== RUN   TestList
=== RUN   TestList/print_configs_of_a_single_kube-context


```

With the check 
```
tejaldesai@@skaffold (add_diag_pkg)$ make test
go test ./pkg/skaffold/...

```

Reply:
Looks like this https://docs.google.com/document/d/1uEUFsUBNbxorsEgtqalmTfmiqd4E6_BVxM5CV-YI7bY/view occupies a bunch of lines on screen